base/complex.jl

```diff
@@ -535,37 +535,42 @@ end
 #     return Complex(abs(iz)/r/2, copysign(r,iz))
 # end
 
-# compute exp(im*theta)
-function cis(theta::Real)
-    s, c = sincos(theta)
-    Complex(c, s)
-end
-
 """
-    cis(z)
+    cis(x)
 
-Return ``\\exp(iz)``.
+Computes a faster version of ``\\exp(i x)``, where `x` is a real or complex number.
```

Comment:
`x` doesn't have to be a number — it could be any type that supports `exp` and `im *`.  For example, #40194 added support for the `cis` of a matrix.

Reply:
How about just
```
Similar to `exp(im*x)`, this computes ``\\exp(i x)``, but is typically faster.